core/common/src/main/java/alluxio/worker/block/io/BlockClient.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block.io;
+
+import java.io.Closeable;
+
+/**
+ * Base interface for block reader/writers.
+ */
+public interface BlockClient extends Closeable {
```

Comment:
I would prefer using `AutoCloseable` unless you think there is a reason we shouldn't? It will make these all compatible with the try-with-resources pattern

Reply:
`Closeable` is also an `AutoCloseable`. So it should be fine to use it in try-with-resources right?